translator/server.py

```diff
@@ -56,7 +56,7 @@ def translate_data(self):
                 translatedText = marian.translate([text])
                 return translatedText[0]
             else:
-                return "Translation Unavailable"
+                return from_code + to_code + "Translation Unavailable"

```

Comment:
I like the idea for this change. I think if we do something like this, it should include the actual languages instead of the ISO language codes so the user knows the translation didn't work and with what languages.